Libplanet.Net.Tests/Protocols/TestTransport.cs

```diff
@@ -244,30 +244,30 @@ async Task DoAddPeersAsync()
                         }
                     }
 
-                    _logger.Verbose("Trying to ping all {PeersNumber} peers.", tasks.Count);
+                    _logger.Verbose("Trying to ping all {PeersNumber} peers", tasks.Count);
                     await Task.WhenAll(tasks);
-                    _logger.Verbose("Update complete.");
+                    _logger.Verbose("Update complete");
                 }
                 catch (DifferentAppProtocolVersionException)
                 {
-                    _logger.Debug("Different version encountered during AddPeersAsync().");
+                    _logger.Debug("Different version encountered during AddPeersAsync()");
```

Comment:
Do we not replace `nameof()` instead of a plain string this time?

Reply:
I guess I missed it since it wasn't caught by regex. I'll fix it on the other branch to be included on a follow-up PR.